trie/node.go

```diff
@@ -0,0 +1,16 @@
+package trie
+
+type node interface {
+	// TODO, will be filled when we implement caching and hashing
+}
```

Comment:
If its not being used we can just remove it for now

Reply:
this is required for compilation